feature-group-definitions/basic-arrays.yml

```diff
@@ -0,0 +1,45 @@
+name: Basic JavaScript arrays
```

Comment:
If I understood correctly, certain consumers (MDN) will only show the baseline banners if the features are recorded web-features. So, if we generally don't deal with basic/old/initial feature bundles, then the MDN pages will not signal baseline high. 

That said, this was my motivation to include them here. Would love to learn more about why it would be good to hold back on these.

Reply:
Yes, you're right, I didn't think that through. This is how we control rollout of the Baseline banner on MDN.

My hesitation is that I don't think we'd want features like this one to be imported into caniuse.com. It has a `shown` field for some features:
https://github.com/Fyrd/caniuse/blob/main/features-json/forms.json

I've sent https://github.com/web-platform-dx/web-features/pull/627 to define a new `legacy` field, and we should encourage consumers that show lists of features to hide such features. We can also use this for features that have "melted into" a parent feature over time, like "Array iteration methods", even if it's split from basic Array.

Would that work?

Reply:
> My hesitation is that I don't think we'd want features like this one to be imported into caniuse.com. It has a `shown` field for some features:
> https://github.com/Fyrd/caniuse/blob/main/features-json/forms.json

But caniuse has "legacy" features imported from MDN as well. Caniuse would want to show the baseline banner on these, too, right? https://caniuse.com/mdn-javascript_grammar_array_literals

Reply:
Resolving as this discussion continues in https://github.com/web-platform-dx/web-features/pull/627

Reply:
> @ddbeck what do you think about a `private: true` or `draft: true` field to hold back whole features from the package?

A no-export `private` field sounds fine to me. I'm not as certain about the do-not-show field.